libcontainer/container_linux.go

```diff
@@ -2237,6 +2296,28 @@ func (c *Container) bootstrapData(cloneFlags uintptr, nsMaps map[configs.Namespa
 		})
 	}
 
+	// Idmap mount sources to open.
+	if it == initStandard && c.shouldSendIdmapSources() {
+		var mounts []byte
+		for _, m := range c.config.Mounts {
+			if m.IsIDMap() {
```

Comment:
for consideration (see other comments)

```suggestion
			if m.IsBind() && m.IsIDMapped() {
```